tests/source/enum-no_trailing_comma.rs

```diff
@@ -0,0 +1,11 @@
+// rustfmt-enum_trailing_comma: false
+
+enum X {
+    A,
+    B,
+}
+
+enum Y {
+    A,
+    B
+}
```

Comment:
could you add tests that have more complicated variants please - tuple-like and struct-like would be good.